src/main/java/net/bitjump/bukkit/subwhitelister/commands/RegisterCommand.java

```diff
@@ -0,0 +1,56 @@
+package net.bitjump.bukkit.subwhitelister.commands;
+
+import java.util.UUID;
+import java.util.logging.Level;
+
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import org.bukkit.plugin.java.JavaPlugin;
+
+/**
+ * Registers a user with their privateKey
+ * @author xtphty
+ *
+ */
+public class RegisterCommand implements CommandExecutor {
+	private static final String USAGE = "Invalid command, use: /register <private key>";
```

Comment:
this is what we are talking about by the way, it should say <login key from destiny.gg> by default and make it configurable from the config
the private key is also a configurable, never exposed thing that is simply sent along with everything to prove that the minecraft server is the one doing the api request
so the authToken is what we are asking the user for, and it is exposed as "login key" in the d.gg profile